File: src/org/jgll/grammar/patterns/ExceptPattern.java
package org.jgll.grammar.patterns;

import java.util.List;

import org.jgll.grammar.Nonterminal;
import org.jgll.grammar.Symbol;

public class ExceptPattern extends AbstractPattern {

	private static final long serialVersionUID = 1L;

	public ExceptPattern(Nonterminal nonteriminal, List<Symbol> parent, int position, List<Symbol> child) {
		super(nonteriminal, parent, position, child);
	}
}

File: src-scala/org/jgll_staged/grammar/patterns/ExceptPattern.scala
package org.jgll_staged.grammar.patterns

import java.util.List
import org.jgll_staged.grammar.Nonterminal
import org.jgll_staged.grammar.Symbol
//remove if not needed
import scala.collection.JavaConversions._

@SerialVersionUID(1L)
class ExceptPattern(nonteriminal: Nonterminal, 
    parent: List[Symbol], 
    position: Int, 
    child: List[Symbol]) extends AbstractPattern(nonteriminal, parent, position, child)
